.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token:
```

Comment:
I guess we don't need this file. Travis knows how to accomplish the job.

Reply:
When you work with **Travis + Coveralls**, all informations about the coverage test are sent with no problems to Coveralls.io.

But, if you work with **CircleCI + Coveralls**, when we execute ```npm run coveralls```, the Coveralls package return the following error: ```Bad response: 422 {"message": "Couldn't find a repository matching this job.", "error": true}```, then, referring to the documentation (that also a bit confused), Coveralls recommends to create the ```.coveralls.yml``` file to refer the repository token and be able to send the coreralls test result to Coveralls.io

Reply:
Indeed. Same as before, let Travis do the job.